mlflow_cratedb/adapter/ddl/cratedb.sql

```diff
@@ -67,7 +67,7 @@ CREATE TABLE IF NOT EXISTS "model_versions" (
    "name" TEXT NOT NULL,
    "version" INTEGER NOT NULL,
    "creation_time" BIGINT,  -- default=get_current_time_millis
-   "last_update_time" BIGINT,  -- default=get_current_time_millis
+   "last_updated_time" BIGINT,  -- default=get_current_time_millis
```

Comment:
This is inconsistent in MLflow across the tables (`last_update_time` vs. `last_updated_time`).

Reply:
Thanks for spotting.